release.py

```diff
@@ -0,0 +1,848 @@
+    updated = re.sub(
+        r'^version: str = \'' + VERSION_REGEX + r'\'$',
+        f"version: str = '{ops_version}'",
+        content,
+        flags=re.MULTILINE,
+    )
+    ops_src_file_path.write_text(updated)
+    logger.info(f'Updated {ops_src_file} to version {ops_version}')
+
+    # pyproject.toml
+    # Update both ops-scenario and ops-tracing versions.
+    ops_pyproject_file = VERSION_FILES['ops/pyproject']
+    ops_pyproject_file_path = Path(ops_pyproject_file)
+    content = ops_pyproject_file_path.read_text()
+    updated = re.sub(
+        r'ops-scenario==' + VERSION_REGEX,
+        f'ops-scenario=={testing_version}',
+        content,
+    )
+    updated = re.sub(
+        r'ops-tracing==' + VERSION_REGEX,
+        f'ops-tracing=={ops_version}',
+        updated,
+    )
+    # Fail if nothing is changed.
+    if content == updated:
+        logger.error(f'No changes made to {ops_pyproject_file}. Check the versions.')
+        exit(1)
+    ops_pyproject_file_path.write_text(updated)
+    logger.info(f'Updated {ops_pyproject_file} to ops {ops_version} testing {testing_version}')
+
+
+def update_testing_version(ops_version: str, testing_version: str):
+    """Update the testing pyproject version."""
+    file = VERSION_FILES['testing']
+    file_path = Path(file)
+    content = file_path.read_text()
+    updated = re.sub(
+        r'version = "' + VERSION_REGEX + '"',
+        f'version = "{testing_version}"',
+        content,
+    )
+    updated = re.sub(
+        r'ops==' + VERSION_REGEX,
+        f'ops=={ops_version}',
+        updated,
+    )
+    # Fail if nothing is changed.
+    if content == updated:
+        logger.error(f'No changes made to {file}. Check the versions.')
+        exit(1)
+    file_path.write_text(updated)
+    logger.info(f'Updated {file} to ops {ops_version} testing {testing_version}')
+
+
+def update_tracing_version(ops_version: str):
+    """Update the tracing pyproject version."""
+    file = VERSION_FILES['tracing']
+    file_path = Path(file)
+    content = file_path.read_text()
+    updated = re.sub(
+        r'version = "' + VERSION_REGEX + '"',
+        f'version = "{ops_version}"',
+        content,
+    )
+    updated = re.sub(
+        r'ops==' + VERSION_REGEX,
+        f'ops=={ops_version}',
+        updated,
+    )
+    # Fail if nothing is changed.
+    if content == updated:
+        logger.error(f'No changes made to {file}. Check the versions.')
+        exit(1)
+    file_path.write_text(updated)
+    logger.info(f'Updated {file} to version {ops_version}')
+
+
+def update_uv_lock():
+    """Update the uv.lock file with the new versions."""
+    subprocess.run(['uv', 'lock'], check=True)  # noqa: S607
+
+
+def parse_scenario_version():
+    """Parse the current scenario version from pyproject.toml."""
+    file = VERSION_FILES['testing']
+    file_path = Path(file)
+    content = file_path.read_text()
+    match = re.search(r'version = "' + VERSION_REGEX + '"', content)
```

Comment:
Does a raw f-string work here?
```suggestion
    match = re.search(rf'version = "{VERSION_REGEX}"', content)
```